packages/instant/src/components/standard_sliding_panel.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import { SlideAnimationState, StandardSlidingPanelContent, StandardSlidingPanelSettings } from '../types';
+
+import { InstallWalletPanelContent } from './install_wallet_panel_content';
+import { SlidingPanel } from './sliding_panel';
+
+export interface StandardSlidingPanelProps extends StandardSlidingPanelSettings {
+    onClose: () => void;
+}
+
+export class StandardSlidingPanel extends React.Component<StandardSlidingPanelProps> {
+    public render(): React.ReactNode {
+        const { animationState, content, onClose } = this.props;
+        return (
+            <SlidingPanel animationState={animationState} onClose={onClose}>
+                {this._getNodeForContent(content)}
+            </SlidingPanel>
+        );
+    }
+    private readonly _getNodeForContent = (content: StandardSlidingPanelContent): React.ReactNode => {
+        switch (content) {
+            case StandardSlidingPanelContent.InstallWallet:
+                return <InstallWalletPanelContent />;
```

Comment:
At first glance this felt a little over-engineered to me -- why not just take the content as a child, instead of sending in a `StandardSlidingPanelContent` and rendering based on that here.  But given how the code works with dispatching the sliding in and out of these, this makes sense

Reply:
I thought about that. Some reasons I went this direction:
1. Would rather have redux actions be pure data when possible.
2. Imagined that all future standard panels would want access to `onClose` and it would be a pain to have to connect them.